proposals/0257-eliminate-vote-fees.md

```diff
@@ -0,0 +1,362 @@
+---
+simd: '0257'
+title: Eliminate vote fees
+authors:
+  - Max Resnick
+  - Ashwin Sekar
+category: Standard
+type: Core
+status: Review
+created: 2024-02-05
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+This proposal removes base fees for voting and replaces them with a system of
+vote inclusion credits paid for by inflation emissions.
+
+The base fee serves two purposes, additional incentive for leaders to include
+votes in their blocks, and protection against denial of service attacks.
+
+The proposal aims to address this with six parts:
+
+1. Base fees for voting will be removed entirely.
+
+2. 25% of the issuance currently dedicated to vote credits will be dedicated to
+   vote inclusion credits for leaders. This is enough to align incentives for
+   leaders to include votes that they receive in a timely manner regardless
+   of how much stake they hold.
+
+3. Validators will earn rewards based on their own absolute performance instead
+   of their relative performance when compared with other validators.
+
+4. Vote transactions will no longer count towards the global CU limit in the
+   block, they will have their own CU limit.
+
+5. To address denial of service attacks via votes,  the cluster will only process
+   votes from 2000 active validators. These validators will be the top 2000
```

Comment:
This will essentially limit number of validators on the cluster to 2000. Without supporting low staked validators it will be hard to onboard new validator providers because they will find it very hard to get this amount of stake. 

This does not guarantee that a high staked validator will not DDOS vote transactions to the cluster. So it seems that you are assuming DDOS can only come from low staked validator which IMO is wrong.